static/js/containers/ProfilePage_test.js

```diff
@@ -256,7 +245,8 @@ describe("ProfilePage", function() {
     [EMPLOYMENT_STEP, 'EmploymentTab'],
   ]) {
     it(`sends the right props to tab components for step ${step}`, () => {
-      return renderComponent(`/profile/${step}`, SUCCESS_ACTIONS).then(([wrapper]) => {
+      let actions = step === PERSONAL_STEP ? SUCCESS_ACTIONS.concat(SET_PROGRAM) : SUCCESS_ACTIONS;
```

Comment:
general comment: it's a bummer that you need to update all of these tests that have nothing to do with your functionality. this has happened to me a ton, and probably everyone else that has changed anything action-related in this codebase. it would be really nice to be able to listen for a _subset_ of actions (ie: "i care that actions A, B, and C were executed, and i'm not really concerned about the other ones") rather than always requiring an exact list of actions all the time